polaris-react/src/components/Tabs/Tabs.tsx

```diff
@@ -480,10 +474,7 @@ export const Tabs = ({
             styles['IconWrap-open'],
         )}
       >
-        <Icon
-          source={polarisSummerEditions2023 ? ChevronDownMinor : CaretDownMinor}
-          tone="subdued"
-        />
+        <Icon source={ChevronDownMinor} tone="subdued" />
```

Comment:
[This PR](https://5d559397bae39100201eedc1-tnpnnmnlft.chromatic.com/?path=/story/all-components-tabs--with-actions)
[Production](https://storybook.polaris.shopify.com/?path=/story/all-components-tabs--with-actions&globals=polarisSummerEditions2023:true)